lms/static/js/learner_dashboard/models/course_card_model.js

```diff
@@ -9,17 +9,34 @@
         function (Backbone) {
         return Backbone.Model.extend({
             initialize: function(data) {
+                this.activeRunMode = null;
                 if (data){
                     this.context = data;
                     //we should populate our model by looking at the run_modes
                     if (data.run_modes.length > 0){
-                        //We only have 1 run mode for this program
-                        this.setActiveRunMode(data.run_modes[0]);
+                        if(data.run_modes.length === 1){
+                            this.setActiveRunMode(data.run_modes[0]);
+                        }
+                        else{
+                            this.setActiveRunMode(
+                                this.getMostRecentRunMode(data.run_modes));
+                        }
                     }
                 }
             },
 
+            getMostRecentRunMode: function(runModes){
```

Comment:
When implemented, shouldn't this method prioritize any run modes in which the user is enrolled? If so, the name of this method is a little misleading. Maybe `getRelevantRunMode` would be better?